dapr-http-client/src/Dapr/Client/HttpClient/PubSub.hs

```diff
@@ -48,7 +48,7 @@ publishTextMessage ::
   Maybe RequestMetadata ->
   m (Either DaprClientError ())
 publishTextMessage config pubsubname topic message =
-  publishMessage config pubsubname topic (ReqBodyBs (T.encodeUtf8 message)) (header "Content-Type" "text/plain")
```

Comment:
`(header "Content-Type" "text/plain")` shouldn't be removed since it's text message instead of JSON.